### ISSUE TITLE
Government of canada Intranet Theme

### ISSUE THREAD

**Issue by hebertg** [wet-boew/wet-boew#8441]
Hi,
We have recently implemented the GOC Intranet theme and have noticed that it has been abandoned.  Would it be possible to confirm if a new one will be available soon or should  we use the Web Experience Toolkit (WET) theme which is the second bullet on this page - http://wet-boew.github.io/v4.0-ci/docs/ref/themesstyle-en.html

Please let me know.
Thanks,
Gaétan

**Comment by duboisp** [comment]
@hebertg it has been abandoned because of lack of available resource to maintain its code base. See #8366

Currently there is **no active work** from my team regarding to release a new one because we would ran in the same issue of a lack of resource to maintain its code base.

Let me know if you or someone you know are interested to maintain and lead the intranet theme.

FYI - The wet-boew theme was primarily designed for the wet-boew website, not as a replacement of the GOC intranet theme.

**Comment by duboisp** [comment]
Yes. Also you can manually update your local version of your theme with the latest released wet-boew files (*.js files). There should be no conflict.

**Comment by LaurentGoderre** [comment]
Except if a plugin CSS changes, they won't get the new CSS which can break the entire plugin.

**Comment by OCHeather** [comment]
So, in a world where folks are using the GoC Intranet Theme (which has now been abandoned), what are some of those groups looking at in terms of options? Looking at future planning, and curious how other groups have chosen to proceed. It doesn't appear that anyone is planning to maintain and lead this theme (unless there has been a development in the last few months.) Thanks!

**Comment by LaurentGoderre** [comment]
@OCHeather see my comment here: https://github.com/wet-boew/wet-boew/issues/8366#issuecomment-422862989
